src/pages/iou/request/step/IOURequestStepParticipants.js

```diff
@@ -1,7 +1,7 @@
 import {useNavigation} from '@react-navigation/native';
+import _ from 'lodash';
```

Comment:
Why are we changing from underscore to lodash here since we dont have any other code change in this file?